numpy/ma/tests/test_old_ma.py

```diff
@@ -289,20 +297,22 @@ def test_testPut(self):
         d = arange(5)
         n = [0, 0, 0, 1, 1]
         m = make_mask(n)
+        m2 = m.copy()
         x = array(d, mask=m)
         self.assertTrue(x[3] is masked)
         self.assertTrue(x[4] is masked)
         x[[1, 4]] = [10, 40]
-        self.assertTrue(x.mask is not m)
```

Comment:
This sort of change worries me, as the original looks deliberate. Is there documentation of this anywhere?